libopenage/config.h.in

```diff
@@ -5,6 +5,8 @@
 #ifndef OPENAGE_CONFIG_H_
 #define OPENAGE_CONFIG_H_
 
+#cmakedefine01 HAVE_THREAD_LOCAL_STORAGE
```

Comment:
Why use the `#cmakedefine01`? The other definitions below use a standard `#define`.